clothing/admin.py

```diff
@@ -22,12 +23,31 @@ class ColorAdmin(admin.ModelAdmin):
     ordering = ['name']
 
 
+class OphaseOrderFilter(admin.SimpleListFilter):
+        """
+        A filter to limit the order overview to a specific ophase
+        """
+        title = 'Ophase'
+        parameter_name = 'ophase'
+
+        def lookups(self, request, model_admin):
+            """
+            Returns a tuple with all ophases
+            """
+            return ((o, o) for o in Ophase.objects.all())
```

Comment:
Why do we need tuples containing the same object twice?

Reply:
Done in a0d66b9